test/specs/elements/Container/Container-test.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+
+import Container from 'src/elements/Container/Container'
+import * as common from 'test/specs/commonTests'
+
+describe('Container', () => {
+  common.isConformant(Container)
+  common.rendersChildren(Container)
+  common.hasUIClassName(Container)
+
+  common.propKeyOnlyToClassName(Container, 'text')
+  common.propKeyOnlyToClassName(Container, 'left', { className: 'aligned' })
+  common.propKeyOnlyToClassName(Container, 'center', { className: 'aligned' })
+  common.propKeyOnlyToClassName(Container, 'right', { className: 'aligned' })
+  common.propKeyOnlyToClassName(Container, 'justified')
+  common.propKeyOnlyToClassName(Container, 'fluid')
+
+  it('renders a <div /> element', () => {
+    shallow(<Container />)
+      .should.have.tagName('div')
+  })
+
+  it('adds the Container class', () => {
+    shallow(<Container />)
+      .should.have.className('container')
+  })
```

Comment:
Sorry, should've been more explicit.  You can keep all tests in this file, just remove this one:

```js
it('adds the Container class', ...
```

All components must include a class that matches the component name.  A `<Container />` must at least include `<div class="container" />`, for example.  Because this is true of all components, it is part of the base conformance test.

The conformance tests use the component's `_meta` object to ensure the className includes the component name.